examples/nlp/language_modeling/conf/megatron_bert_config.yaml

```diff
@@ -2,7 +2,7 @@ name: megatron_bert
 restore_from_path: null # used when starting from a .nemo file
 
 trainer:
-  devices: 2
```

Comment:
Why make changes to this file ?

Reply:
default conversion script uses this yaml file and use 1 gpu by default. and other models like https://github.com/NVIDIA/NeMo/blob/36a31c03497985c8f2165d4f9be788b574443391/examples/nlp/language_modeling/conf/megatron_gpt_config.yaml uses 1 gpu as well. Is there a reason to do 2?